### ISSUE TITLE
Where is the code?

### ISSUE THREAD

**Issue by jeankvd** [liuervehc/nxmtp#24]
All i see is a readme

**Comment by Catarax** [comment]
https://github.com/liuervehc/nxmtp/releases

**Comment by ShockDW** [comment]
> https://github.com/liuervehc/nxmtp/releases

The "source code" for both releases contains only the readme.md, same as the git itself.  There's no actual source code published here.

**Comment by qwoodmansee** [comment]
Agreed, would be nice to get source so we can update the build for Atmo 11. @liuervehc could you push a branch with the source code? Totally understand if you don't want to handle updating, but having a starting point for updating would be great
